freeipa/src/main/java/com/sequenceiq/freeipa/entity/Stack.java

```diff
@@ -502,7 +502,7 @@ public CcmConnectivityParameters getCcmParameters() {
         if (ccmParameters != null && ccmParameters.getRaw() != null) {
             return JsonUtil.readValueOpt(ccmParameters.getRaw(), CcmConnectivityParameters.class).orElse(null);
         }
-        return null;
+        return new CcmConnectivityParameters();
```

Comment:
doesn't this change have any side effect?

Reply:
this is the only place where this value is used, and it won't modify its output
https://github.com/hortonworks/cloudbreak/blob/c0460ed0f58966d80d74fe14716e9809617bfd65/freeipa/src/main/java/com/sequenceiq/freeipa/service/image/userdata/UserDataBuilder.java#L83-L97

even its test was written with this default value instead of null
https://github.com/hortonworks/cloudbreak/blob/c0460ed0f58966d80d74fe14716e9809617bfd65/freeipa/src/test/java/com/sequenceiq/freeipa/service/image/userdata/UserDataBuilderTest.java#L179

Reply:
thanks for checking